api/models/botfile.go

```diff
@@ -0,0 +1,35 @@
+package models
+
+import (
+  "strconv"
+  "path"
+  "time"
+)
+
+// Go, y u no enum
```

Comment:
:sob:

Reply:
I guess you can technically do this but it just adds so little enum-ness and more friction in usage.
```go
type BotStatus string

const (
		PENDING  BotStatus = "pending"
		UPLOADED BotStatus = "uploaded"
		RUNNABLE BotStatus = "runnable"
		FAILED   BotStatus = "failed"
)
```

Reply:
noted but too lazy